File: AQ_LIB/resources/api/RExamples/Dates/M_DATE_YEARFRACTION.R
dyn.load("MLIB_CLIENT_API")
source("MLIB_CLIENT_API.R")
getLoadedDLLs()
setUpMLIB("","")
meUtilityVersion()

fromDate <- c("20190601")
toDate   <- c("20190901")
dayCount <- c("ACT/ACT")
includeLast <- c(TRUE)

yearfract <- meDateYearFraction( fromDate, toDate, dayCount, includeLast)
yearfract

yearfract <- meDateYearFraction( fromDate, toDate, dayCount )
yearfract

File: AQ_LIB/resources/api/PythonExamples/Dates/M_DATE_YEARFRACTION.py
#Import the MLIB CLIENT API
import MLIB_CLIENT_API as m

#Load Calendars and Config Files
m.setUpMLIB("","")

#Get the MLIB Version
m.meUtilityVersion()

fromDate = "20190601"
toDate   = "20190901"
dayCount = "ACT/ACT"
includeLast = True

yearfract = m.meDateYearFraction( fromDate, toDate, dayCount, includeLast)
print yearfract

yearfract = m.meDateYearFraction( fromDate, toDate, dayCount )
print yearfract